care_o_bot_robot/package.xml

```diff
@@ -10,17 +10,17 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <run_depend>cob_extern</run_depend>
-  <run_depend>cob_common</run_depend>
-  <run_depend>cob_calibration_data</run_depend>
-  <run_depend>cob_environments</run_depend>
-  <run_depend>cob_substitute</run_depend>
-  <run_depend>cob_driver</run_depend>
-  <run_depend>schunk_modular_robotics</run_depend>
-  <run_depend>ros_canopen</run_depend>
-  <run_depend>cob_control</run_depend>
-  <run_depend>cob_command_tools</run_depend>
-  <run_depend>cob_robots</run_depend>
+  <exec_depend>cob_extern</exec_depend>
+  <exec_depend>cob_common</exec_depend>
+  <exec_depend>cob_calibration_data</exec_depend>
+  <exec_depend>cob_environments</exec_depend>
+  <exec_depend>cob_substitute</exec_depend>
+  <exec_depend>cob_driver</exec_depend>
+  <exec_depend>schunk_modular_robotics</exec_depend>
+  <exec_depend>ros_canopen</exec_depend>
+  <exec_depend>cob_control</exec_depend>
+  <exec_depend>cob_command_tools</exec_depend>
+  <exec_depend>cob_robots</exec_depend>
```

Comment:
all these packages can be replaced by just
```
<exec_depend>cob_bringup</exec_depend>
```
(in case the dependencies are set correctly -> installing only this package would be the test whether our dependencies are correct :wink:

Additionally we could add
```
<exec_depend>cob_navigation</exec_depend>
<exec_depend>cob_manipulation</exec_depend>
```